src/libsystemd/sd-bus/bus-objects.c

```diff
@@ -1554,13 +1554,17 @@ void bus_node_gc(sd_bus *b, struct node *n) {
         free(n);
 }
 
-static int bus_find_parent_object_manager(sd_bus *bus, struct node **out, const char *path) {
+static int bus_find_parent_object_manager(sd_bus *bus, struct node **out, const char *path, bool* path_has_object_manager) {
         struct node *n;
 
         assert(bus);
         assert(path);
 
         n = hashmap_get(bus->nodes, path);
+
+        if (n)
+                *path_has_object_manager = !LIST_IS_EMPTY(n->object_managers);
```

Comment:
add `assert(path_has_object_manager) if new contract suggests it must be non-Null

Reply:
Done. Also `LIST_IS_EMPTY` has been removed on recent master so I removed it as well.